ui/admin/app/components/form/credential/json/secret-editor/index.js

```diff
@@ -0,0 +1,29 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import { computed } from '@ember/object';
+
+export default class FormCredentialJsonSecretEditorComponent extends Component {
+  // =actions
+  @tracked allowEditing = false;
+
+  @action
+  allowEdit() {
+    this.allowEditing = true;
+  }
+
+  @computed('args.{isNew,isEditing}')
```

Comment:
Are these `computed` needed? `allowEditing` is already tracked and I thought `args` were automatically tracked with glimmer components?

Reply:
Yep that is right. I was having issues with `allowEditing` triggering a render after it was changed but I think it was due to using computed vs leaving it off and letting tracked do its job. Fixed now.